src/core/components/RootNavLayout.css

```diff
@@ -0,0 +1,39 @@
+.root-layout-container {
+  position: fixed;
+  overflow: auto;
+  width: 100%;
+  display: grid;
+  grid-template-rows: 7.5vh 84vh 8.5vh;
```

Comment:
Let's set the height of header and nav-bar areas with px or rem units, as it should not change on different screens sizes, unless we adjust it with  @media
The middle area could be set to 1fr as it should fill all the available for it area

Reply:
Ok! I Canged header and footer to 3.75 and 4.19 rem respectively.